env/env.go

```diff
@@ -8,31 +8,39 @@ import (
 )
 
 type Env struct {
-	LookupEnv func(key string) (string, bool)
-	Getenv    func(key string) string
-	Setenv    func(key, value string) error
-	Unsetenv  func(key string) error
-	Environ   func() []string
-	Map       map[string][]string
+	RootDirMap map[string][]string
+	vars       map[string]string
+}
+
+func varsFromEnviron(environ []string, removeKey func(string) bool) map[string]string {
+	vars := make(map[string]string)
+	for _, kv := range environ {
+		parts := strings.Split(kv, "=")
```

Comment:
Use `strings.SplitN` to guarantee that the string is only split on the first `=`